apps/userdashboard/routers.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import routers
+
+from adhocracy4.api.routers import CustomRouterMixin
+
+
+class ModerationDetailRouterMixin(CustomRouterMixin):
+
+    prefix_regex = (
+        r'userdashboard/moderation/(?P<project_pk>[\d]+)/{prefix}'
```

Comment:
What is the prefix for?

Reply:
Its to include the project_pk in the routes, so you can for example fetch /userdashboard/moderation/1/userclassifications list. Then project_pk=1 is in kwargs of the viewset. Its like its done for example in OrganisationDefaultRouter or ModuleDefaultRouter.

Reply:
And the prefix itself after the project pk? That's the UserClassification or AIClassification, then, right?

Reply:
Oh sorry, that didnt answer your question :see_no_evil: But yes exactly, its the prefix of the urls registered for this router, so userclassifications or aiclassifications